platform/ioscross/README.md

```diff
@@ -0,0 +1,52 @@
+Cross compiling iOS from Linux
+==============================
+
+This directory contains scripts for compiling a iOS version 
+of Csound from Linux.  These scripts were developed using Ubuntu.
+
+**[Please ensure you have read and understood the Xcode license
+terms first before using it.](https://www.apple.com/legal/sla/docs/xcode.pdf)**
```

Comment:
I'm pretty sure this says you must develop for Apple machines using Apple products and this cross compilation (or use of their SDKS on non-Apple machines) is breaking the EULA. That's one thing for a dev to do it as a hobby project but for an organisation like Csound it might mean something else. Can you explain if this is legally gray or otherwise?

Reply:
There are large open source organizations that allow cross-compiling for Apple machines.
The Godot project has [documentation](https://docs.godotengine.org/en/stable/contributing/development/compiling/cross-compiling_for_ios_on_linux.html) on how to cross compile their project for ios using cctools-port.  Some of the changes here are based on their documentation.
They do have a [disclaimer](https://docs.godotengine.org/en/stable/contributing/development/compiling/cross-compiling_for_ios_on_linux.html#disclaimer) in their documentation that specifies the following:

> While it is possible to compile for iOS on a Linux environment, Apple is very restrictive about the tools to be used (especially hardware-wise), allowing pretty much only their products to be used for development. So this is not official. However, in 2010 Apple said they relaxed some of the [App Store review guidelines](https://developer.apple.com/app-store/review/guidelines/) to allow any tool to be used, as long as the resulting binary does not download any code, which means it should be OK to use the procedure described here and cross-compiling the binary.